### ISSUE TITLE
Styling of hyperlinks in content relies on color only (a11y)

### ISSUE THREAD

**Issue by JocelynDelalande** [BookStackApp/BookStack#4939]
### Describe the Bug

In default style, hyperlinks are distinguishable from regular text only by color, which [is not enough and problematic from an accessibility standpoint](https://www.a11yproject.com/checklist/#ensure-that-links-are-recognizable-as-links) (think colorblind).

Note that in info boxes, there is underlining on links.

Underlining on hover is not enough.

![image](https://github.com/BookStackApp/BookStack/assets/429633/2860b6ce-5344-4064-8c44-98686b20135d)

### Steps to Reproduce

![image](https://github.com/BookStackApp/BookStack/assets/429633/d931b754-2504-40e7-8b30-4034826e5d89)

### Expected Behaviour

underline links in content column.

### Screenshots or Additional Context

Workaround (as bookstack admin)

Add a `<style>` tag in Settings / Customization with the following content:

 ```css
 #tinymce main.content-wrap.card a,
  .page-content a {
  text-decoration: underline;
}
```

### Exact BookStack Version

v23.08.3

**Comment by ssddanbrown** [comment]
Thanks @JocelynDelalande, sounds like a sensible improvement to make, have assigned it to the next feature release.

### Dev Notes

- Check various scenarios with linked non-text content (including variations with mixed non-text/text), across browsers.

### Doc Upates

- Upgrade notice regarding change to default content link styles, since this could be a significant change to user content.

**Comment by szabeszg** [comment]
After installing BookStack for the very fist time, my first style addition was to add underlines to links, so it is a welcome change for sure!

**Comment by ssddanbrown** [comment]
I've now addressed this in content via 0f6cb9ed840a61fe211321ae0d2d0cd0bafadfbd, to be part of the next feature release.
There are various sceanrios where we also have links in the UI itself, although many of those can be context specific and not where underlines may be productive (or they already have other hints to a link), but those areas can be addressed as needed based upon feedback and/or improvement as areas evolve.

**Comment by JocelynDelalande** [comment]
Excellent, thanks :)
